src/js/components/Select/SelectContainer.js

```diff
@@ -42,6 +42,34 @@ const ClearButton = forwardRef(
     const align = position !== 'bottom' ? 'start' : 'center';
     const buttonLabel = label || `Clear ${name || 'selection'}`;
     const { passThemeFlag } = useThemeValue();
+    const buttonKind = theme.select.clear?.button;
+
+    if (buttonKind) {
+      // New structure when `kind` is defined
+      const containerProps = theme.select.clear?.container || {};
+      const textProps = theme.select.clear?.text || {};
+
+      return (
+        <Box flex="grow" {...containerProps}>
+          <StyledButton
+            a11yTitle={`${buttonLabel}. Or, press ${
+              position === 'bottom' ? 'shift tab' : 'down arrow'
+            } to move to select options`}
+            fill="horizontal"
+            justify="start"
+            ref={ref}
+            onClick={onClear}
+            kind={theme.select.clear?.button}
+            label={<Text {...textProps}>{buttonLabel}</Text>}
```

Comment:
```suggestion
            label={buttonLabel}
```
If someone if going the "select.clear.button" route, I'm wondering if we should just use the text styles defined there and ignore theme.select.clear.text.

@jcfilben thoughts?

Reply:
I like the conditional

Reply:
cool Ill change

Reply:
done